nvme.c

```diff
@@ -1553,6 +1555,8 @@ static int get_log(int argc, char **argv, struct command *cmd, struct plugin *pl
 		__u64 lpo;
 		__u8  lsp;
 		__u8  uuid_index;
+		__u8  csi;
+		__u8  ot;
```

Comment:
Maybe this should just be an int as 307142b ("nvme: add get log page 2.0 spec fields") and define 'ot' just as bool, the same way we do for 'rae'

Reply:
I changed the type of the 'ol' to bool.